script.js

```diff
@@ -0,0 +1,137 @@
+const userInput = document.querySelector(".s-text-input");
+const searchBtn = document.querySelector(".s-btn");
+const ul = document.querySelector("ul");
+const div = document.querySelector("div");
+let isPrev = false;
+
+
+
+function deletePrevList() {
+  const allLinks = document.querySelectorAll("li");
+  ul.innerHTML = "";
+}
+
+function progressBar(width = 0) {
```

Comment:
this function is written also in company.js. use another file for shared functions. DRY